flow/connectors/bigquery/bigquery.go

```diff
@@ -972,6 +977,22 @@ func (c *BigQueryConnector) SetupNormalizedTables(
 			idx++
 		}
 
+		if req.SoftDeleteColName != "" {
+			columns = append(columns, &bigquery.FieldSchema{
```

Comment:
Put a `+ 2` on the capacity when allocating slice. Better to overestimate by 2 elements than underestimate & reallocate